solidity/contracts/deposit/DepositUtils.sol

```diff
@@ -447,24 +447,58 @@ library DepositUtils {
     ///                     is this deposit's TDT holder.
     /// @param _redeemer    The assumed owner of the deposit's TDT.
     /// @return             The amount in TBTC needed to redeem the deposit.
-    function getOwnerRedemptionTbtcRequirement(DepositUtils.Deposit storage _d, address _redeemer) internal view returns(uint256) {
-        // The TDT owner only ever owes fee adjustments.
-        return computeRedemptionFeeAdjustment(_d, _redeemer);
+    function getOwnerRedemptionTbtcRequirement(
+        DepositUtils.Deposit storage _d,
+        address _redeemer
+    ) internal view returns(uint256 owedToDeposit, uint256 owedToRedeemer) {
```

Comment:
I'm not sure why `getOwnerRedemptionTbtcRequirement` would return anything other than the amount owed to the deposit.

Reply:
Ah yes, forgot to remove this